File: Pypy-C++/example_main.py
#!/usr/bin/pypy

import cpp_example

if __name__ == '__main__':
    a = cpp_example.Example('teszt-A')
    b = cpp_example.Example('teszt-B')

    print a.Get()
    print b.Get()
    print a.Get()
    print a.Get()
    print a.Get()
    print a.Get()
    print b.Get()

File: C/example_main.lua
#!/usr/bin/luajit

require ("c_example")

function main()
    a = c_example.Example('teszt-A')
    b = c_example.Example('teszt-B')

    print (c_example.Get(a))
    print (c_example.Get(b))
    print (c_example.Get(a))
    print (c_example.Get(a))
    print (c_example.Get(a))
    print (c_example.Get(a))
    print (c_example.Get(b))
end

main()
